archetypes/helidon/src/main/archetype/common/observability.xml

```diff
@@ -57,9 +57,13 @@ curl -H 'Accept: application/json' -X GET http://localhost:8080/metrics
 ]]></value>
                                     </list>
                                     <list key="dependencies">
+                                        <map>
```

Comment:
Please remove `helidon-webserver-observe-metrics` and `helidon-metrics-system-meters` dependencies from the **MP** pom.xml as they are brought in transitively (by `helidon-microprofile-metrics`). I verified this with Tim. Leave them in SE.